sdk/metric/internal/sum.go

```diff
@@ -164,6 +158,43 @@ func (s *cumulativeSum[N]) Aggregation() metricdata.Aggregation {
 	return out
 }
 
+type precomputedValue[N int64 | float64] struct {
+	// measured is the value directly measured.
+	measured N
+	// filtered is the sum of values from spatially aggregations.
```

Comment:
am I correct in thinking that only one of [filtered, measured] will end up being set?  Is there a case where both are set?

Reply:
If an observation is made that doesn't filter any attributes, `measured` is _set_. If an observation is made that does filter out attributes, `filtered` will be _added_ to. There may be a callback(s) that does both, so both may be used.